### PR TITLE
Fold button HTML

### PR DESCRIPTION
**Pull request by sophianicolas**

### CODE REVIEW

File: Sprint_4/source/foldButton_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from flask import Flask, request, render_template
+from CardGames import *
+
+app = Flask(__name__)
+
```
Comment (erinyooey) [inline Sprint_4/source/foldButton_test.py:5]: Requirements are satisfied as you created an html button that triggers the fold! I don't see any logic error in calling the fold.html.

**Review by erinyooey — APPROVED** [review]

**Review by robcas12 — APPROVED** [review]
Requirement Satisfaction: Meets requirements per trello board
Logic Errors: No major logic error issues.
Test Coverage: Good test coverage
Style Guides: no major style guide issues

File: Sprint_4/source/foldButton_test.py (hunk 1 above)
Comment (sophianicolas) [reply Sprint_4/source/foldButton_test.py:5]: Thanks for looking through and letting me know! It currently doesn't trigger any actual action.

**Comment by sophianicolas** [comment]
Thanks for looking through and letting me know!

**Pull request closed without merging**